src/components/tile/_tile.scss

```diff
@@ -0,0 +1,168 @@
+//-----------------------------
+// Tiles
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/css--typography';
+
+@include exports('tile') {
+  .bx--tile-container {
+    width: 100%;
+
+    .bx--grid {
+      margin: 0;
+      padding: 0;
+    }
+
+    .bx--col {
```

Comment:
I think this is the same as above - these are demo styles that shouldn't be in the main tiles component